web_monitoring/differs.py

```diff
@@ -5,3 +5,7 @@ def dummy(a, b, **options):
 
 def compare_length(a, b):
```

Comment:
This may lead to false positives in the situation when two byte-strings have the same length but they are, in fact, different.